app/src/main/java/com/github/libretube/helpers/NavigationHelper.kt

```diff
@@ -110,6 +111,19 @@ object NavigationHelper {
         }
     }
 
+    /**
+     * Open a large, zoomable image preview
+     */
+    suspend fun openImagePreview(context: Context, url: String) {
+        val image = ImageHelper.getImage(context, url) ?: return
+
+        // we can't pass it as a bundle argument, since the max size of them is 500KB
+        ZoomableImageActivity.bitmap = image
```

Comment:
The image URL could be passed instead. That way, the size shouldn't be an issue.